eradiate/tests/system/test_compare_rami4atm_rami_onedim.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import eradiate
+
+from eradiate.scenes.biosphere import DiscreteCanopy
+from eradiate.units import unit_registry as ureg
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("reflectance", [0.0, 0.5, 1.0])
+@pytest.mark.parametrize("sza", [0.0, 30.0, 60.0])
+def test_compare_rami4atm_onedim(mode_mono_double, reflectance, sza):
+    """
+    Compare Rami4ATMExperiment with OneDimExperiment (``path``)
+    ============================================================
+
+    This test compares the results of a OneDimExperiment with a corresponding
+    Rami4ATMExperiment run. To achieve that, the Rami4ATMExperiment is set up without an explicit
+    canopy and a basic atmosphere.
+
+    Rationale
+    ---------
+
+    Both experiments are set up with the same surface and atmosphere:
+
+    * Geometry: a square surface with unit size and a Lambertian BRDF with
+      reflectance :math:`\rho \in [0.0, 0.5, 1.0]`.
+    * Illumination: Directional illumination with a zenith angle
+      :math:`\theta \in [0.0, 30.0, 60.0]°`.
+    * Atmosphere: Molecular atmosphere using the ussa1976 profile
+    * Sensor: Distant reflectance measure covering a plane (91 angular points,
+      10000 sample per pixel)
+      We take an odd number of angular points to ensure that the special value
+      :math:`\theta = 0°` is included.
+
+    Expected behaviour
+    ------------------
+
+    The results of the two runs must agree within a relative tolerance of 1e-3
+    """
+
+    onedim = eradiate.experiments.OneDimExperiment(
+        surface={"type": "lambertian", "reflectance": reflectance},
+        illumination={"type": "directional", "zenith": sza, "azimuth": 0.0},
+        atmosphere={
+            "type": "heterogeneous",
+            "molecular_atmosphere": {
+                "type": "molecular",
+                "has_absorption": False,
+                "has_scattering": True,
+            },
+        },
+        measures={
+            "type": "distant_reflectance",
+            "id": "measure",
+            "film_resolution": (21, 1),
+            "spp": 1e4,
+        },
+    )
+
+    r4a = eradiate.experiments.Rami4ATMExperiment(
+        surface={"type": "lambertian", "reflectance": reflectance},
+        illumination={"type": "directional", "zenith": sza, "azimuth": 0.0},
+        atmosphere={
+            "type": "heterogeneous",
+            "molecular_atmosphere": {
+                "type": "molecular",
+                "has_absorption": False,
+                "has_scattering": True,
+            },
+        },
+        measures={
+            "type": "distant_reflectance",
+            "id": "measure",
+            "film_resolution": (21, 1),
+            "spp": 1e4,
+        },
+        canopy=None,
+    )
+
+    onedim.run()
+    r4a.run()
+
+    assert np.allclose(
+        onedim.results["measure"].lo.values,
+        r4a.results["measure"].lo.values,
+        atol=1e-6,
+    )
+
```

Comment:
Would you mind adding a figure for the test report? See *e.g.* `test_onedim_lambertian_brf.py`.